### DISCUSSION TITLE
Enable the use of vscode snippets in the prompt input

### DISCUSSION THREAD

**Discussion by mholtzhausen** [cline/cline#768] [Feature Requests]
I've been using vscode snippets to organize my frequently-used prompts for use across multiple projects and in order to share with my team. This works great with Github Copilot, as I can trigger snippet insertion straight into the prompt input. When I try this with Cline, it dumps the snippet into the nearest open file. 

If there is some other mechanism that is widely used for managing a library of prompts, I'd love to hear, but this is the flow the currently works the best for me - would be great if it were supported by this fantastic AI extension.

**Comment by GLStephen** [comment]
This seems like the best way to bridge the prompt library while leveraging existing functionality.
